src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -726,59 +674,52 @@ def _set_dsync_params(self,
-                Alternatively, this can be the pool uuid.
-            src_cont (TestContainer, optional): the source container.
-                Alternatively, this can be the container uuid.
+            src_pool (TestPool, optional): the source pool or uuid.
+            src_cont (TestContainer, optional): the source cont or uuid.
             dst_type (str): how to interpret the dst params.
                 Must be in PARAM_TYPES.
             dst_path (str): posix-style destination path.
                 For containers, this is relative to the container root.
-            dst_pool (TestPool, optional): the destination pool.
-                Alternatively, this can be the pool uuid.
-            dst_cont (TestContainer, optional): the destination container.
-                Alternatively, this can be the container uuid.
-
+            dst_pool (TestPool, optional): the destination pool or uuid.
+            dst_cont (TestContainer, optional): the destination cont or uuid.
         """
         # First, initialize a new dsync command
-        self.dsync_cmd = Dsync(self.hostlist_clients)
+        self.dsync_cmd = DsyncCommand(self.hostlist_clients, self.workdir)
         self.dsync_cmd.get_params(self)
 
         # Set the source params
         if src_type == "POSIX":
-            self.dsync_cmd.set_dsync_params(
-                src=str(src_path))
+            self.dsync_cmd.set_params(
+                src_path=str(src_path))
         elif src_type == "DAOS_UUID":
-            param = self._format_daos_path(src_pool, src_cont, src_path)
-            self.dsync_cmd.set_dsync_params(
-                src=param)
+            self.dsync_cmd.set_params(
+                src_path=self.format_daos_path(src_pool, src_cont, src_path))
         elif src_type == "DAOS_UNS":
             if src_cont:
                 if src_path == "/":
-                    self.dsync_cmd.set_dsync_params(
-                        src=src_cont.path.value)
+                    self.dsync_cmd.set_params(
+                        src_path=src_cont.path.value)
                 else:
-                    self.dsync_cmd.set_dsync_params(
-                        prefix=src_cont.path.value,
-                        src=src_cont.path.value + src_path)
+                    self.dsync_cmd.set_params(
+                        daos_prefix=src_cont.path.value,
+                        src_path=src_cont.path.value + src_path)
 
         # Set the destination params
         if dst_type == "POSIX":
-            self.dsync_cmd.set_dsync_params(
-                dst=dst_path)
+            self.dsync_cmd.set_params(
+                dst_path=str(dst_path))
         elif dst_type == "DAOS_UUID":
-            param = self._format_daos_path(dst_pool, dst_cont, dst_path)
-            self.dsync_cmd.set_dsync_params(
-                dst=param)
+            self.dsync_cmd.set_params(
+                dst_path=self.format_daos_path(dst_pool, dst_cont, dst_path))
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member